src/client/extension.ts

```diff
@@ -77,10 +77,14 @@ function startREPLProc(context: ExtensionContext, config: settings.ISettings, pw
             }
             client.sendRequest(EvaluateRequestMessage, evaluateArgs)
 
-            await proc.showTerminalIfVisible();
+            await proc.showTerminalIfNotVisible();
         }
 
 
+      let cmdShowTerminal = commands.registerCommand("powershell.showTerminal", async () =>  proc.showTerminal());
```

Comment:
```suggestion
      let cmdShowTerminal = commands.registerCommand("powershell.showTerminal", proc.showTerminal);
```
I _think_ this can be simplified